pmoves/services/common/nats_service_listener.py

```
@@ -0,0 +1,468 @@
+    nc: NATS | None = None
+    backoff = 1.0
+    max_backoff = 30.0
+
+    while True:
+        try:
+            nc = NATS()
+            await nc.connect(nats_url)
+
+            await nc.publish(SERVICE_ANNOUNCE_SUBJECT, announcement.to_json().encode())
+            logger.info(f"Service announcement published: {slug} at {url}")
+            return True
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)